src/db/index.js

```diff
@@ -1,83 +1,99 @@
 const { JsonDB } = require("node-json-db");
 const { Config } = require("node-json-db/dist/lib/JsonDBConfig");
-const AsyncLock = require("async-lock");
-
-// Default hmwk-db.json filename.
-const DB_FILENAME = "hmwk-db";
+const Redis = require("ioredis");
+const JSONCache = require("redis-json");
+require("dotenv").config();
 
 const RESEED_PATH_PREFIX = "/reseed";
 const TRACKER_PATH_PREFIX = "/tracker";
 const SUBMIT_PATH_PREFIX = "/submit";
 
-// Implements concurrent DB supporting thread-safe read/write/delete operations
-class ConcurrentDB {
+// Decide between Redis or JsonDB.
+const USE_REDIS = process.env.USE_REDIS === "true";
```

Comment:
Will this environment variable be set automatically by Heroku when it starts our app?

Reply:
No, you'll need to set this env var